.github/workflows/tests.yaml

```diff
@@ -23,7 +23,7 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, windows-latest, macos-latest]
-        python-version: ["3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.9", "3.10", "3.11"]
```

Comment:
Maybe we should exclude one osx build since they are slowing us down significantly.

Reply:
IIUC the environment is identical to 3.10. I suggest to skip OSX+3.10 then